src/core_modules/capture-core/components/D2Form/index.js

```diff
@@ -1,4 +1,3 @@
 // @flow
-
 export { asyncHandlerActionTypes, asyncUpdateFieldEpic } from './asyncHandlerHOC';
-
+export { D2Form } from './D2Form.container';
```

Comment:
thought of exporting the file from the index as we decided we would like to be doing